README.md

```diff
@@ -4,27 +4,29 @@ This model archive contains code used for deep learning and hydrodynamic modelin
 
 You will need to have [Anaconda](https://docs.anaconda.com/anaconda/install/index.html), [R](https://cran.r-project.org/), and [mamba](https://mamba.readthedocs.io/en/latest/installation.html) installed on your computer to run this pipeline.
 
-To retrain a new model run all steps below (1-8). If you only want to analyze results from the manuscript, run steps 1-4, then use steps 7-8 for analysis. Model ouptut used for the manuscript is located in `03_model/out/Manuscript_Results`.
+To retrain a new model run all steps 1-6 below.
+
+If you want to reproduce the analysis of the results from the manuscript, run steps 1-4 to prepare the environment and input datasets, then use steps 7-8 to analyze the results contained in `03_model\out\Run_Manuscript_Results`.
+
+If you would like to analyze the results of your own model run that you produced by running steps 1-6, you may need to manually reconfigure some of the scripts used in steps 7-8 of the pipeline, as there are elements that are hard-coded for the manuscript run.
 
 ## Prepare your environment
 
 1) Download the model archive and unzip its contents.
 2) Navigate into the root directory of the archive and use [mamba](https://mamba.readthedocs.io/en/latest/installation.html) to create the environment using `mamba env create -f environment.yaml`.
 3) Activate your conda environment with `conda activate drb_estuary_salinity`.
 
-## To train the model:  
+## To train a new model:  
 
 4) Run `snakemake -s Snakefile_fetch_munge -j` (-j runs the job on the available number of cpus cores, use -j 2 for fewer) to fetch and munge the data inputs used in the model. You might have to rerun the same command if an error that pops up, this is because snakemake doesn't run rules in the necessary order sometimes.
-5) Open the file `03_model/model_config.yaml` and adjust modeling parameters. Change the run_id to whatever you want to name your run of the model (e.g. Test_Run). **Note**: If you do not change the `run_id` to something other than 'Run_Manuscript_Results', the manuscript results will be overwritten in the next step.
+5) Open the file `03_model/model_config.yaml` and adjust modeling parameters. Change the run_id to whatever you want to name your run of the model (e.g. Test_Run).
 6) Run `snakemake -s Snakefile_run_ml_model run_replicates -j` to train your model. Your model results will be written to `03_model/out/{run_id}/` where `{run_id}` is the run_id specified in `03_model/model_config.yaml`.
 
-## To analyze model output and produce figures:
+## To analyze model output and produce figures for the modeling run used in the associated manuscript (contained in 03_model\out\Run_Manuscript_Results):
 
```

Comment:
I suggest changing this to 
## To analyze model output and produce figures:
To reproduce figures in the associated manuscript complete steps 7 and 8 with `{run_id}` set to `Run_Manuscript_Results`, to produce figures for a newly trained model complete steps 7 and 8 with `{run_id}` set to the name of the new model run directory

Reply:
What about something like this (to reiterate that you need to go edit the scripts):

> To reproduce figures in the associated manuscript, complete steps 7 and 8 with {run_id} in `model_config_Run_Manuscript_Results.yaml` set to `Run_Manuscript_Results`. To produce figures for a newly trained model, complete steps 7 and 8 with {run_id} set to the name of the new model run directory. You will also need to manually review the scripts used in the Snakefiles listed below to reflect certain modeling configuration decisions you have made (such as the number of replicates).

Reply:
That sounds good to me!